modules/dataverse-parent/pom.xml

```diff
@@ -441,8 +447,8 @@
             <snapshots>
                 <enabled>true</enabled>
             </snapshots>
-        </repository>
-        -->
+            </repository>
+	 -->
     </repositories>
     
```

Comment:
@landreev tests were failing as of https://jenkins.dataverse.org/job/IQSS-Dataverse-Develop-PR/job/PR-11810/7/testReport/

<img width="1828" height="1186" alt="Screenshot 2025-10-24 at 9 35 08 AM" src="https://github.com/user-attachments/assets/7ac13580-101c-42f3-8edf-7bad0b19d91d" />

Then I merged the latest from develop in. The run has been kicked off: https://jenkins.dataverse.org/job/IQSS-Dataverse-Develop-PR/job/PR-11810/8/

Fingers crossed!